src/generate_syscalls_perf.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+SYSCALLS_CSV=$1
+SYSCALLS_PERF_TEMPLATE=$2
+
+cat $SYSCALLS_CSV | grep -v '^#' | nl -ba -s, -v0 | \
+    sed -e '/^#/d' \
```

Comment:
Is this `sed -e '/^#/d` needed since you have the `grep -v  '^#'` above?

Reply:
correct, dropped it